### PR TITLE
Requested ASCN Updates

### PR DESCRIPTION
**Pull request by averyniceday**
Fix cBioPortal/cbioportal# (see https://help.github.com/en/articles/closing-issues-using-keywords)

Describe changes proposed in this pull request:
- Rename several columns
- Separate Indeterminate vs NA  (no facets analysis) samples
- Change coloring for subclonal samples

## Checks
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

## Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [Giphy CAPTURE](https://giphy.com/apps/giphycapture) or [Peek](https://github.com/phw/peek)

## Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). It can help to figure out who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them either through slack or by assigning them as a reviewer on the PR

### CODE REVIEW

File: src/shared/components/mutationTable/column/ascnCopyNumber/ASCNCopyNumberColumnFormatter.tsx [hunk 1]
```diff
@@ -155,18 +156,24 @@ export default class ASCNCopyNumberColumnFormatter {
                 'totalCopyNumber'
             )
                 ? mutation.alleleSpecificCopyNumber.totalCopyNumber.toString()
+                : hasASCNProperty(mutation, 'ascnMethod')
```
Comment (averyniceday) [inline src/shared/components/mutationTable/column/ascnCopyNumber/ASCNCopyNumberColumnFormatter.tsx:159]: differentiate between cases with `no FACETS analysis` vs `indeterminate results during FACETS analysis`

**Comment by averyniceday** [comment]
@zhx828 still need to update the e2e tests and rebase both the feature branch and my dev branch to the front of master. But I'm going to be out the next week so if you get the chance can you give a quick review on the added code changes. Pretty minor stuff -- just hiding some columns and changing the colors.

File: src/shared/components/mutationTable/column/expectedAltCopies/ExpectedAltCopiesElement.tsx [hunk 2]
```diff
@@ -19,26 +25,96 @@ export const ExpectedAltCopiesElementTooltip: React.FunctionComponent<{
...
     sampleManager?: SampleManager | null;
 }> = props => {
-    return (
+    return props.expectedAltCopiesValue === 'NA' ? (
+        <span>
+            <MutantIntegerCopyNumberIcon
+                expectedAltCopiesValue={props.expectedAltCopiesValue}
+            />
+        </span>
+    ) : (
```
Comment (sheridancbio) [inline src/shared/components/mutationTable/column/expectedAltCopies/ExpectedAltCopiesElement.tsx:109]: this is far from the "?" operator ... would this be better as
```
if (condition)
    return <component>
else
    return <component with tooltip>
```  
?

**Review by sheridancbio — APPROVED** [review]
I think this is good as is. I point to several uses of the ternary operator, which I am not a big fan of ... so I suggest considering alternatives in a few places. But work looks good.

**Comment by averyniceday** [comment]
already cherry-picked into the main PR that was merged

**Pull request closed without merging**